app/views/coronavirus_landing_page/_live_stream.html.erb

```diff
@@ -14,11 +14,21 @@
   <div class="govuk-grid-column-two-thirds">
     <%= render "govuk_publishing_components/components/govspeak", {
     } do %>
-      <p class="govuk-body">
-        <a href="<%= url %>" class="govuk-link covid__page-header-link">
+      <div class="covid__video-placeholder">
+        <div class="covid__video-logo">Live</div>
+        <p class="govuk-body">
+          <a href="/help/cookies" class="govuk-link covid__page-header-link covid__bold">Change your cookie settings</a>
```

Comment:
We should move these over to the yaml file

Reply:
I'm not sure we should - I don't think there's going to be any need for content devs to change these.

Reply:
Have now done - see related PR in the description.